### ISSUE TITLE
No Profiles Are Available

### ISSUE THREAD

**Issue by ColeRichardson** [hexparrot/mineos-node#300]
hi i recently installed mineos plugin on freenas 11.2, when i try to load profiles none show up and i get the error message in the title, i checked network connectivity by pinging google.com from the jail's shell in freenas. seems fine to me as no packets were lost and response time was quick, ive also tried uninstalling and reinstalling mineos. ive also done a bunch of google and havent been able to find anything relevant. help please! 
![mineosprofilesnotloadiung](https://user-images.githubusercontent.com/30738295/65372082-53483b00-dc39-11e9-9c8c-5b95cac5e75b.PNG)

**Comment by hexparrot** [comment]
Seems this problem existed because the webui wasn't gracefully handling when URIs for profiles were broken or _temporarily failing_.  A commit has been made to address this, so updating the webui should fix it.  I presume this means updating the mineos plugin (which I hope automatically is the latest commit?).

b52b72efc30a929cde769b792be4a9cef6ecd36c

**Comment by rshaklef0rd** [comment]
I too am having the same issue. I am familiar with the web ui, but have never used SSH or FTP to get into my servers files. How do I go about applying the commit? I really don't know what terms to search, and am completely new to OpenBSD/FreeNAS, so I am not familiar with the syntax yet. Can you point me in the direction of any "how-to" threads for applying this fix?

**Comment by hexparrot** [comment]
No need to apply a commit specifically—it all gets rolled into normal webui update steps:

http://minecraft.codeemo.com/mineoswiki/index.php?title=Updating_the_Webui

**Comment by rshaklef0rd** [comment]
So I went to the link that you sent. I have been accessing the shell through the Jail in FreeNAS. Am I able to login as root from there? Or will I need to SSH into the server? (Like I said, I am unfamiliar with the syntax, or how it is applied) I had a couple servers running, and kind of dealt with snapshots for a bit, then settled on a release, set it and forgot about it, until we went to put the latest release on, and now I am feeling like I need to learn a lot more, but need to figure out how to start.
